fs/fs.go

```diff
@@ -862,23 +862,21 @@ func FileExists(fs Fs, remote string) (bool, error) {
 	return true, nil
 }
 
-// CalculateModifyWindow works out modify window for Fses passed in -
-// sets Config.ModifyWindow
-//
-// This is the largest modify window of all the fses in use, and the
-// user configured value
-func CalculateModifyWindow(fss ...Fs) {
+// GetModifyWindow calculates the maximum modify window between the given Fses
+// and the Config.ModifyWindow parameter.
+func GetModifyWindow(fss ...Info) time.Duration {
+	window := Config.ModifyWindow
 	for _, f := range fss {
 		if f != nil {
 			precision := f.Precision()
-			if precision > Config.ModifyWindow {
-				Config.ModifyWindow = precision
-			}
 			if precision == ModTimeNotSupported {
-				Infof(f, "Modify window not supported")
-				return
+				Debugf(f, "Modify window not supported")
```

Comment:
We should probably remove this Debug as it will show for every Copy operation

Reply:
Oh yeah, good point. Done and merged!